packages/buckets/src/buckets.ts

```diff
@@ -103,22 +138,22 @@ export class Buckets extends BucketsGrpcClient {
    *
    * const open = async (auth: UserAuth, name: string) => {
    *     const buckets = Buckets.withUserAuth(auth)
-   *     await buckets.open(name)
-   *     return buckets
+   *     const { root, threadID } = await buckets.getOrInitBucket(name)
+   *     return { buckets, root, threadID }
    * }
    * ```
    */
-  async open(
+  async getOrInitBucket(
```

Comment:
snuck this in so that if we add the api into go buckets, i can just replace this internal logic with the endpoint. deprecates `open`